prow/plugins/trigger/pull-request.go

```diff
@@ -106,6 +106,46 @@ func handlePR(c Client, trigger plugins.Trigger, pr github.PullRequestEvent) err
 				return buildAll(c, &pr.PullRequest, pr.GUID, trigger.ElideSkippedContexts)
 			}
 		}
+		if pr.Label.Name == labels.OkToTest {
+			// Ticket # 13975. When Ok-To-Test label is added directly.
+			labelAddedBy := pr.Sender.Name
+			// If label is added by the bot, then skip it.
+			// this could also be possible if someone goes the traditional way of adding /ok-to-test then /test-all comments
```

Comment:
Tests will start as soon as we `/ok-to-test`

Reply:
I picked that info from the description in the issue:

> Normally people do this with an ok-to-test comment, which will add the label and trigger tests. However people can also just add the ok-to-test label. This currently requires them to also leave a /test all comment to trigger testing.

Also, you want me to change the comment?

Reply:
We could leave a comment like:


```
// When the bot adds the label from an /ok-to-test command,
// we will trigger tests based on the comment event and do not
// need to trigger them here from the label, as well
```